src/AutoRest.CSharp.V3/Generation/Writers/JsonCodeWriterExtensions.cs

```diff
@@ -134,6 +134,12 @@ public static void ToSerializeCall(this CodeWriter writer, JsonSerialization ser
                             writer.Append($"({name}")
                                 .AppendNullableValue(implementationType);
 
+                            // TODO: Turn a char into a string so it can be passed to WriteStringValue
```

Comment:
Can you add an overload to https://github.com/Azure/autorest.csharp/blob/feature/v3/src/assets/Azure.Core.Shared/Utf8JsonWriterExtensions.cs#L12